### ISSUE TITLE
Decide on the string to be read on the help tooltip

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/mozilla-vpn-client#7299]
We should define what should be read when the user focuses on the help tooltip, while using the TalkBack feature.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/VPN-5094)
┆Reporter: Santiago Andrigo

**Comment by data-sync-user** [comment]
➤ Chris Rooney commented:

Santiago Andrigo this is going to be audited along with the rest of the to the talkback strings but will be done by EOW

**Comment by data-sync-user** [comment]
➤ Chris Rooney commented:

Santiago Andrigo “Get help” should probably suffice. TalkBack will say “double tap to activate” afterward

**Comment by data-sync-user** [comment]
➤ Santiago Andrigo commented:

SGTM! Thx
